config/initializers/authentication.rb

```diff
@@ -0,0 +1,21 @@
+# Check for authentication method and copy data over if necessary (ENV variable
+# to skip if necessary)
+unless ENV['SKIP_AUTH_INIT']
```

Comment:
Only when you have CAS authentication enabled and a database where at least one user is missing their `cas_login` parameters.